src/components/common/Directors.vue

```diff
@@ -456,6 +456,14 @@
                     >
                       <span>Remove</span>
                     </v-btn>
+                    <v-btn color="error"
+                     class="reset-edit-btn"
+                     v-show="!isNew(director) && editFormShowHide.showName"
```

Comment:
I think we want to show the button for a new director but leave it disabled. This is to show users where the button lives if they edit a director.

Reply:
Interesting... that seems weird to me because the UI is different for editing a new director and a director name change. The new director has a 'Remove' btn, i don't understand showing a btn there that they would never use.

Reply:
Lets talk about this offline, i think it might be a design question!

Reply:
Maybe you're right. But this is a Scott question.